app/posts/apply-for-teacher-training/2021-09-22-degree-grade-requirement-warning-message.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Degree grade requirement warning message
+description: Warning candidates when they do not meet the degree grade requirement.
+date: 2021-09-22
+---
+
+One of the reasons why providers reject candidates is because they do not have the right degree grade (for example 2:1 or above).
+
+Each course has a different minimum degree requirement. While some courses accept any honours degree, most ask for a minimum grade.
+
+This is particularly challenging for candidates with a third-class degree.
+
+We realised that we could help candidates by:
+
+* showing the minimum degree requirement for each course (on Find)
+* warning them if they’ve chosen a course that may not accept their degree grade
+
+## Showing minimum degree requirements on Find
+
+We added a line to the course summary in the Find search results.
+
+![Course summary on Find with a line about the minimum degree requirement.](course-summary-find.png)
```

Comment:
Could link to the 2 design history entries for Find about this?

* https://bat-design-history.netlify.app/find-teacher-training/standardised-entry-requirement-descriptions/
* https://bat-design-history.netlify.app/find-teacher-training/degree-requirement-filter/

There's also one for Publish: https://bat-design-history.netlify.app/publish-teacher-training-courses/degree-entry-requirements/